dashboard/client/average_response_time/average_response_time.js

```diff
@@ -15,12 +15,13 @@ Template.averageResponseTime.onRendered(function () {
 
     if (elasticsearchData) {
       // Get aggregations from Elasticsearch data
-      const aggregatedData = elasticsearchData.aggregations.avg_response_time.buckets;
+      const aggregatedData = elasticsearchData.aggregations.requests_over_time.buckets;
```

Comment:
Because ElasticSearch calculates the average response time for *each date range*, I mean, it is based on aggregated data "request_over_time". Each bucket "request_over_time" containsvalue "avg_response_time".

Please fetch data and look at the response